src/object/obj_utils.c

```diff
@@ -114,8 +114,12 @@ obj_query_merge_recx(struct daos_oclass_attr *oca, daos_unit_oid_t oid, daos_key
 	 * replica ext from parity shard needs not to convert.
 	 */
 	tmp_end = DAOS_RECX_END(tmp_recx);
-	D_DEBUG(DB_IO, "shard %d/%u get recx "DF_U64" "DF_U64"\n",
-		oid.id_shard, tgt_off, tmp_recx.rx_idx, tmp_recx.rx_nr);
+
+	D_DEBUG(DB_IO, "Before merge: "DF_UOID", stripe_nr %u, cell_nr %u, tgt_off %u, from %s, %s"
```

Comment:
LIne 10 in this file should be 

#define D_LOGFAC        DD_FAC(object)

I've been playing around with this trying to debug and hit that issue.